### ISSUE TITLE
Start, Stop Worker from CLi in Linux

### ISSUE THREAD

**Issue by fractalnode** [Codaone/DEXBot#703]
I would like to be able to start and stop single worker from commandline in Linux
e.g
cli.py worker1 stop
cli.py worker1 start

**Comment by PermieBTS** [comment]
Postponed until cli refactor along with other cli improvements

**Comment by thehapax** [comment]
@PermieBTS @MarkoPaasila Do we have any cli configuration packages that were previously considered and evaluated? I recall it was discussed.  Curating a list of what you'd like here would help facilitate the process of replacing Whiptail. Whiptail is not well maintained and limited in functionality.

**Comment by bitfag** [comment]
Good idea, I like it, should be considered when splitting core and frontends.
